scripts/update_schemas.py

```diff
@@ -23,6 +23,26 @@
  * github organization.
  ***************************************************************/"""
 
+# OEM schemas
+oem_schema_names = [
```

Comment:
Should these be sorted alphabetically to make it easy to see what is already in the list and to mimic the order of the rest of the schemas?

Reply:
The current change is to keep the same order of the Oem schemas as they were before.

However, it can be sorted by adding oem filenames before sort of csdl filenames.
Perhaps we can do it in a separate PR (and upstream commit).

```
--- a/scripts/update_schemas.py
+++ b/scripts/update_schemas.py
@@ -159,7 +159,6 @@ json_schema_files = OrderedDict(
     sorted(json_schema_files.items(), key=lambda x: SchemaVersion(x[0]))
 )

-csdl_filenames.sort(key=SchemaVersion)

 # Create oem filenames - from oem json names
 oem_csdl_filenames = []
@@ -169,6 +168,8 @@ for filename in oem_schema_names:
 # Append Oem csdl files
 csdl_filenames += oem_csdl_filenames

+csdl_filenames.sort(key=SchemaVersion)

```